btr-bagit-profile.json

```diff
@@ -43,17 +43,12 @@
             "required": false,
             "recommended": true,
             "description": "Can be the same as source_organization recommended when not the same as source"
-        },
-        "Data-Export-Info": {"required": false},
-        "Data-Export-Agent": {"required": false},
-        "Data-Export-Agent-Version": {"required": false},
-        "Data-Export-Format": {"required": false},
-        "Data-Export-Format-Version": {"required": false},
-        "Relationship-Info": {"required": false}
+        }
     },
     "Manifests-Required": [],
-    "Manifests-Supported": [
+    "Manifests-Allowed": [
```

Comment:
Other tags that list acceptable/allowed value sets use the "Accept-" prefix, e.g. `Accept-Serialization` and `Accept-BagIt-Version`. Is there a reason to not follow that pattern and use `Accept-Manifests` (and `Accept-Tag-Manifests` below)?

Reply:
We're sticking to the name patterns used in the primary bagit-profiles repo at https://github.com/bagit-profiles/bagit-profiles. Take a look at the comments from jscancella on my pull request at https://github.com/bagit-profiles/bagit-profiles/pull/20. tl;dr is they don't want to change tag names and break existing validators.